sensor_stack/sparton/src/sparton.py

```diff
@@ -336,6 +336,7 @@ def new_msg_format():
 
 
     quaternion = tf.transformations.quaternion_from_euler(roll, pitch, yaw)
+    print 'Roll='+str(roll)+' Pitch='+str(pitch)+' Yaw='+str(yaw)
```

Comment:
You can remove this line now. It's not required anymore, it will simply clutter stdout. `rostopic echo` will provide a much pleasant-er view of the huge amount of data that will be dumped.